easybuild/easyconfigs/m/MUSCLE3/MUSCLE3-0.7.2-foss-2023b.eb

```diff
@@ -0,0 +1,105 @@
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'MUSCLE3'
+version = '0.7.2'
+
+homepage = 'https://muscle3.readthedocs.io/en/latest/index.html'
+
+description = """MUSCLE3 allows connecting multiple simulation models
+ together into a multiscale simulation. Simulation models can be as
+ simple as a single Python file, or as complex as a combination of
+ multiple separate simulation codes written in C++ or Fortran, and
+ running on an HPC machine."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://github.com/multiscale/%(namelower)s/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['9596899f4a576d4ca80cc54c4cd458aac25ef0038a97843c76cec60a7105f779']
+
+builddependencies = [
+    ('googletest', '1.14.0'),
+]
+
+dependencies = [
+    ('msgpack-c', '6.0.0'),
```

Comment:
The foss installation is failing as it cannot find a msgpack-c with foss2023b or below. If you add the msgpack-c recipe with GCC-13.2.0 this part will work

Reply:
I've added the `msgpack-c/6.0.0` EB files for the intel and foss 2023b toolchains.